Index: admin/src/main/java/com/info/admin/controller/problemlibrary/ProblemLibraryController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/java/com/info/admin/controller/problemlibrary/ProblemLibraryController.java	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/java/com/info/admin/controller/problemlibrary/ProblemLibraryController.java	(date 1542769958424)
@@ -88,7 +88,7 @@
     @RequestMapping(value="/addOrEdit",method={RequestMethod.GET,RequestMethod.POST})
     public String addOrEdit(HttpServletRequest request,String supplierId,Model model){
         try{
-            if(null != supplierId && StringUtils.isNotBlank(supplierId)){
+            if(StringUtils.isNotBlank(supplierId)){
                 //根据id查询系统用户
                 ProblemLibrary problemLibrary = service.getProblemLibraryById(supplierId);
                 model.addAttribute("problemLibrary", problemLibrary);
@@ -119,9 +119,16 @@
             if (null == entity) {
                 return new JsonResult(JsonResultCode.FAILURE, "请输入数据", "");
             }
-
+            String[] inspectUsers = request.getParameterValues("inspectUsers[]");
+            if(inspectUsers != null && inspectUsers.length > 0){
+                entity.setInspectUser(StringUtils.join(inspectUsers,","));
+            }
+            String[] rectifyUsers = request.getParameterValues("rectifyUsers[]");
+            if(rectifyUsers != null && rectifyUsers.length> 0){
+                entity.setRectifyUser(StringUtils.join(rectifyUsers,","));
+            }
             // 通过id来判断是新增还是修改
-            if (null != entity.getSupplierId() && StringUtils.isNotBlank(entity.getSupplierId())) {
+            if (StringUtils.isNotBlank(entity.getSupplierId())) {
                 result = service.update(entity);
             } else {
                 entity.setDeleteFlag(0L);
Index: admin/src/main/java/com/info/admin/controller/releaseinfo/ReleaseInfoController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/java/com/info/admin/controller/releaseinfo/ReleaseInfoController.java	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/java/com/info/admin/controller/releaseinfo/ReleaseInfoController.java	(date 1542769945010)
@@ -206,7 +206,7 @@
     @RequestMapping(value="/addOrEditUser",method={RequestMethod.GET,RequestMethod.POST})
     public String addOrEditUser(HttpServletRequest request,String releaseId,Model model){
         try{
-            if(null != releaseId && StringUtils.isNotBlank(releaseId)){
+            if(StringUtils.isNotBlank(releaseId)){
                 //根据id查询系统用户
                 ReleaseInfo releaseInfo = service.getReleaseInfoById(releaseId);
                 model.addAttribute("releaseInfo", releaseInfo);
@@ -271,7 +271,7 @@
                 entity.setReceiveUserCn(StringUtils.join(receiveUserCns,","));
             }
             // 通过id来判断是新增还是修改
-            if (null != entity.getReleaseId() && StringUtils.isNotBlank(entity.getReleaseId())) {
+            if ( StringUtils.isNotBlank(entity.getReleaseId())) {
                 result = service.update(entity);
             } else {
                 entity.setDeleteFlag(0L);
Index: admin/src/main/java/com/info/admin/entity/ProblemLibrary.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/java/com/info/admin/entity/ProblemLibrary.java	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/java/com/info/admin/entity/ProblemLibrary.java	(date 1542770351507)
@@ -5,183 +5,203 @@
 
 /**
  * 问题库 实体类
- * @author ysh  
+ *
+ * @author ysh
  * @date 2018-11-19 18:06:32
  */
 public class ProblemLibrary implements Serializable {
-	private static final long serialVersionUID = 1L;
-	 /*****主键*****/
-	 private String supplierId;
-	 /*****创建时间*****/
-	 private Date createTime;
-	 /*****创建时间 日常的字符串*****/
-	 private String createTimeStr;
-	 /*****创建人编号*****/
-	 private Long createUser;
-	 /*****删除标记*****/
-	 private Long deleteFlag;
-	 /*****修改时间*****/
-	 private Date updateTime;
-	 /*****修改时间 日常的字符串*****/
-	 private String updateTimeStr;
-	 /*****排序号*****/
-	 private Long seq;
-	 /*****标题*****/
-	 private String title;
-	 /*****问题类型*****/
-	 private Long type;
-	 /*****检查内容*****/
-	 private String inspectContent;
-	 /*****检查人编号*****/
-	 private String inspectUser;
-	 /*****整改人编号*****/
-	 private String rectifyUser;
-	 /*****整改时间*****/
-	 private Date rectifyTime;
-	 /*****整改时间 日常的字符串*****/
-	 private String rectifyTimeStr;
+    private static final long serialVersionUID = 1L;
+    /*****主键*****/
+    private String supplierId;
+    /*****创建时间*****/
+    private Date createTime;
+    /*****创建时间 日常的字符串*****/
+    private String createTimeStr;
+    /*****创建人编号*****/
+    private Long createUser;
+    /*****删除标记*****/
+    private Long deleteFlag;
+    /*****修改时间*****/
+    private Date updateTime;
+    /*****修改时间 日常的字符串*****/
+    private String updateTimeStr;
+    /*****排序号*****/
+    private Long seq;
+    /*****标题*****/
+    private String title;
+    /*****问题类型*****/
+    private Long type;
+    /*****检查内容*****/
+    private String inspectContent;
+    /*****检查人编号*****/
+    private String inspectUser;
+    /*****整改人编号*****/
+    private String rectifyUser;
+    /*****检查人编号*****/
+    private String inspectUserCn;
+    /*****整改人编号*****/
+    private String rectifyUserCn;
+    /*****整改时间*****/
+    private Date rectifyTime;
+    /*****整改时间 日常的字符串*****/
+    private String rectifyTimeStr;
 
 
-	 public String getSupplierId() {
-		 return supplierId;
-	 }
+    public String getSupplierId() {
+        return supplierId;
+    }
 
-	 public void setSupplierId(String supplierId) {
-		 this.supplierId = supplierId;
-	 }
+    public void setSupplierId(String supplierId) {
+        this.supplierId = supplierId;
+    }
 
-	 public Date getCreateTime() {
-	 	 if(org.apache.commons.lang.StringUtils.isNotBlank(createTimeStr) && createTime == null){
-			 createTime = com.info.admin.utils.DateUtil.stringToDate(createTimeStr);
-		 }
-		 return createTime;
-	 }
+    public Date getCreateTime() {
+        if (org.apache.commons.lang.StringUtils.isNotBlank(createTimeStr) && createTime == null) {
+            createTime = com.info.admin.utils.DateUtil.stringToDate(createTimeStr);
+        }
+        return createTime;
+    }
 
-	 public void setCreateTime(Date createTime) {
-		 this.createTime = createTime;
-	 }
-	 
-	 public String getCreateTimeStr() {
-		 return createTimeStr;
-	 }
+    public void setCreateTime(Date createTime) {
+        this.createTime = createTime;
+    }
+
+    public String getCreateTimeStr() {
+        return createTimeStr;
+    }
 
-	 public void setCreateTimeStr(String createTimeStr) {
-		 this.createTimeStr = createTimeStr;
-		 if(org.apache.commons.lang.StringUtils.isNotBlank(createTimeStr) && createTime == null){
-			this.createTime = com.info.admin.utils.DateUtil.stringToDate(createTimeStr);
-		 }
-	 }
+    public void setCreateTimeStr(String createTimeStr) {
+        this.createTimeStr = createTimeStr;
+        if (org.apache.commons.lang.StringUtils.isNotBlank(createTimeStr) && createTime == null) {
+            this.createTime = com.info.admin.utils.DateUtil.stringToDate(createTimeStr);
+        }
+    }
 
-	 public Long getCreateUser() {
-		 return createUser;
-	 }
+    public Long getCreateUser() {
+        return createUser;
+    }
 
-	 public void setCreateUser(Long createUser) {
-		 this.createUser = createUser;
-	 }
+    public void setCreateUser(Long createUser) {
+        this.createUser = createUser;
+    }
 
-	 public Long getDeleteFlag() {
-		 return deleteFlag;
-	 }
+    public Long getDeleteFlag() {
+        return deleteFlag;
+    }
 
-	 public void setDeleteFlag(Long deleteFlag) {
-		 this.deleteFlag = deleteFlag;
-	 }
+    public void setDeleteFlag(Long deleteFlag) {
+        this.deleteFlag = deleteFlag;
+    }
 
-	 public Date getUpdateTime() {
-	 	 if(org.apache.commons.lang.StringUtils.isNotBlank(updateTimeStr) && updateTime == null){
-			 updateTime = com.info.admin.utils.DateUtil.stringToDate(updateTimeStr);
-		 }
-		 return updateTime;
-	 }
+    public Date getUpdateTime() {
+        if (org.apache.commons.lang.StringUtils.isNotBlank(updateTimeStr) && updateTime == null) {
+            updateTime = com.info.admin.utils.DateUtil.stringToDate(updateTimeStr);
+        }
+        return updateTime;
+    }
 
-	 public void setUpdateTime(Date updateTime) {
-		 this.updateTime = updateTime;
-	 }
-	 
-	 public String getUpdateTimeStr() {
-		 return updateTimeStr;
-	 }
+    public void setUpdateTime(Date updateTime) {
+        this.updateTime = updateTime;
+    }
+
+    public String getUpdateTimeStr() {
+        return updateTimeStr;
+    }
 
-	 public void setUpdateTimeStr(String updateTimeStr) {
-		 this.updateTimeStr = updateTimeStr;
-		 if(org.apache.commons.lang.StringUtils.isNotBlank(updateTimeStr) && updateTime == null){
-			this.updateTime = com.info.admin.utils.DateUtil.stringToDate(updateTimeStr);
-		 }
-	 }
+    public void setUpdateTimeStr(String updateTimeStr) {
+        this.updateTimeStr = updateTimeStr;
+        if (org.apache.commons.lang.StringUtils.isNotBlank(updateTimeStr) && updateTime == null) {
+            this.updateTime = com.info.admin.utils.DateUtil.stringToDate(updateTimeStr);
+        }
+    }
 
-	 public Long getSeq() {
-		 return seq;
-	 }
+    public Long getSeq() {
+        return seq;
+    }
 
-	 public void setSeq(Long seq) {
-		 this.seq = seq;
-	 }
+    public void setSeq(Long seq) {
+        this.seq = seq;
+    }
 
-	 public String getTitle() {
-		 return title;
-	 }
+    public String getTitle() {
+        return title;
+    }
 
-	 public void setTitle(String title) {
-		 this.title = title;
-	 }
+    public void setTitle(String title) {
+        this.title = title;
+    }
 
-	 public Long getType() {
-		 return type;
-	 }
+    public Long getType() {
+        return type;
+    }
 
-	 public void setType(Long type) {
-		 this.type = type;
-	 }
+    public void setType(Long type) {
+        this.type = type;
+    }
 
-	 public String getInspectContent() {
-		 return inspectContent;
-	 }
+    public String getInspectContent() {
+        return inspectContent;
+    }
 
-	 public void setInspectContent(String inspectContent) {
-		 this.inspectContent = inspectContent;
-	 }
+    public void setInspectContent(String inspectContent) {
+        this.inspectContent = inspectContent;
+    }
 
-	 public String getInspectUser() {
-		 return inspectUser;
-	 }
+    public String getInspectUser() {
+        return inspectUser;
+    }
 
-	 public void setInspectUser(String inspectUser) {
-		 this.inspectUser = inspectUser;
-	 }
+    public void setInspectUser(String inspectUser) {
+        this.inspectUser = inspectUser;
+    }
 
-	 public String getRectifyUser() {
-		 return rectifyUser;
-	 }
+    public String getRectifyUser() {
+        return rectifyUser;
+    }
 
-	 public void setRectifyUser(String rectifyUser) {
-		 this.rectifyUser = rectifyUser;
-	 }
+    public void setRectifyUser(String rectifyUser) {
+        this.rectifyUser = rectifyUser;
+    }
 
-	 public Date getRectifyTime() {
-	 	 if(org.apache.commons.lang.StringUtils.isNotBlank(rectifyTimeStr) && rectifyTime == null){
-			 rectifyTime = com.info.admin.utils.DateUtil.stringToDate(rectifyTimeStr);
-		 }
-		 return rectifyTime;
-	 }
+    public Date getRectifyTime() {
+        if (org.apache.commons.lang.StringUtils.isNotBlank(rectifyTimeStr) && rectifyTime == null) {
+            rectifyTime = com.info.admin.utils.DateUtil.stringToDate(rectifyTimeStr);
+        }
+        return rectifyTime;
+    }
 
-	 public void setRectifyTime(Date rectifyTime) {
-		 this.rectifyTime = rectifyTime;
-	 }
-	 
-	 public String getRectifyTimeStr() {
-		 return rectifyTimeStr;
-	 }
+    public void setRectifyTime(Date rectifyTime) {
+        this.rectifyTime = rectifyTime;
+    }
+
+    public String getRectifyTimeStr() {
+        return rectifyTimeStr;
+    }
 
-	 public void setRectifyTimeStr(String rectifyTimeStr) {
-		 this.rectifyTimeStr = rectifyTimeStr;
-		 if(org.apache.commons.lang.StringUtils.isNotBlank(rectifyTimeStr) && rectifyTime == null){
-			this.rectifyTime = com.info.admin.utils.DateUtil.stringToDate(rectifyTimeStr);
-		 }
-	 }
+    public void setRectifyTimeStr(String rectifyTimeStr) {
+        this.rectifyTimeStr = rectifyTimeStr;
+        if (org.apache.commons.lang.StringUtils.isNotBlank(rectifyTimeStr) && rectifyTime == null) {
+            this.rectifyTime = com.info.admin.utils.DateUtil.stringToDate(rectifyTimeStr);
+        }
+    }
 
-	 
-	public ProblemLibrary() {
-		super();
-	}
+    public String getInspectUserCn() {
+        return inspectUserCn;
+    }
+
+    public void setInspectUserCn(String inspectUserCn) {
+        this.inspectUserCn = inspectUserCn;
+    }
+
+    public String getRectifyUserCn() {
+        return rectifyUserCn;
+    }
+
+    public void setRectifyUserCn(String rectifyUserCn) {
+        this.rectifyUserCn = rectifyUserCn;
+    }
+
+    public ProblemLibrary() {
+        super();
+    }
 }
Index: admin/src/main/java/com/info/admin/vo/ProblemLibraryVo.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/java/com/info/admin/vo/ProblemLibraryVo.java	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/java/com/info/admin/vo/ProblemLibraryVo.java	(date 1542770397233)
@@ -5,133 +5,154 @@
 
 /**
  * 问题库实体类
+ *
  * @author administrator  2018-11-19 18:06:32
  */
 public class ProblemLibraryVo implements Serializable {
-	private static final long serialVersionUID = 1L;
-	 /*****主键*****/ 
-	 private String supplierId;
-	 /*****创建时间*****/ 
-	 private Date createTime;
-	 /*****创建人编号*****/ 
-	 private Long createUser;
-	 /*****删除标记*****/ 
-	 private Long deleteFlag;
-	 /*****修改时间*****/ 
-	 private Date updateTime;
-	 /*****排序号*****/ 
-	 private Long seq;
-	 /*****标题*****/ 
-	 private String title;
-	 /*****问题类型*****/ 
-	 private Long type;
-	 /*****检查内容*****/ 
-	 private String inspectContent;
-	 /*****检查人编号*****/ 
-	 private String inspectUser;
-	 /*****整改人编号*****/ 
-	 private String rectifyUser;
-	 /*****整改时间*****/ 
-	 private Date rectifyTime;
+    private static final long serialVersionUID = 1L;
+    /*****主键*****/
+    private String supplierId;
+    /*****创建时间*****/
+    private Date createTime;
+    /*****创建人编号*****/
+    private Long createUser;
+    /*****删除标记*****/
+    private Long deleteFlag;
+    /*****修改时间*****/
+    private Date updateTime;
+    /*****排序号*****/
+    private Long seq;
+    /*****标题*****/
+    private String title;
+    /*****问题类型*****/
+    private Long type;
+    /*****检查内容*****/
+    private String inspectContent;
+    /*****检查人编号*****/
+    private String inspectUser;
+    /*****整改人编号*****/
+    private String rectifyUser;
+    /*****检查人编号*****/
+    private String inspectUserCn;
+    /*****整改人编号*****/
+    private String rectifyUserCn;
+    /*****整改时间*****/
+    private Date rectifyTime;
 
 
-	 public String getSupplierId() {
-		 return supplierId;
-	 }
+    public String getSupplierId() {
+        return supplierId;
+    }
 
-	 public void setSupplierId(String supplierId) {
-		 this.supplierId = supplierId;
-	 }
+    public void setSupplierId(String supplierId) {
+        this.supplierId = supplierId;
+    }
 
-	 public Date getCreateTime() {
-		 return createTime;
-	 }
+    public Date getCreateTime() {
+        return createTime;
+    }
 
-	 public void setCreateTime(Date createTime) {
-		 this.createTime = createTime;
-	 }
+    public void setCreateTime(Date createTime) {
+        this.createTime = createTime;
+    }
 
-	 public Long getCreateUser() {
-		 return createUser;
-	 }
+    public Long getCreateUser() {
+        return createUser;
+    }
 
-	 public void setCreateUser(Long createUser) {
-		 this.createUser = createUser;
-	 }
+    public void setCreateUser(Long createUser) {
+        this.createUser = createUser;
+    }
 
-	 public Long getDeleteFlag() {
-		 return deleteFlag;
-	 }
+    public Long getDeleteFlag() {
+        return deleteFlag;
+    }
 
-	 public void setDeleteFlag(Long deleteFlag) {
-		 this.deleteFlag = deleteFlag;
-	 }
+    public void setDeleteFlag(Long deleteFlag) {
+        this.deleteFlag = deleteFlag;
+    }
 
-	 public Date getUpdateTime() {
-		 return updateTime;
-	 }
+    public Date getUpdateTime() {
+        return updateTime;
+    }
 
-	 public void setUpdateTime(Date updateTime) {
-		 this.updateTime = updateTime;
-	 }
+    public void setUpdateTime(Date updateTime) {
+        this.updateTime = updateTime;
+    }
 
-	 public Long getSeq() {
-		 return seq;
-	 }
+    public Long getSeq() {
+        return seq;
+    }
 
-	 public void setSeq(Long seq) {
-		 this.seq = seq;
-	 }
+    public void setSeq(Long seq) {
+        this.seq = seq;
+    }
 
-	 public String getTitle() {
-		 return title;
-	 }
+    public String getTitle() {
+        return title;
+    }
 
-	 public void setTitle(String title) {
-		 this.title = title;
-	 }
+    public void setTitle(String title) {
+        this.title = title;
+    }
 
-	 public Long getType() {
-		 return type;
-	 }
+    public Long getType() {
+        return type;
+    }
 
-	 public void setType(Long type) {
-		 this.type = type;
-	 }
+    public void setType(Long type) {
+        this.type = type;
+    }
 
-	 public String getInspectContent() {
-		 return inspectContent;
-	 }
+    public String getInspectContent() {
+        return inspectContent;
+    }
 
-	 public void setInspectContent(String inspectContent) {
-		 this.inspectContent = inspectContent;
-	 }
+    public void setInspectContent(String inspectContent) {
+        this.inspectContent = inspectContent;
+    }
 
-	 public String getInspectUser() {
-		 return inspectUser;
-	 }
+    public String getInspectUser() {
+        return inspectUser;
+    }
 
-	 public void setInspectUser(String inspectUser) {
-		 this.inspectUser = inspectUser;
-	 }
+    public void setInspectUser(String inspectUser) {
+        this.inspectUser = inspectUser;
+    }
 
-	 public String getRectifyUser() {
-		 return rectifyUser;
-	 }
+    public String getRectifyUser() {
+        return rectifyUser;
+    }
 
-	 public void setRectifyUser(String rectifyUser) {
-		 this.rectifyUser = rectifyUser;
-	 }
+    public void setRectifyUser(String rectifyUser) {
+        this.rectifyUser = rectifyUser;
+    }
 
-	 public Date getRectifyTime() {
-		 return rectifyTime;
-	 }
+    public String getInspectUserCn() {
+        return inspectUserCn;
+    }
+
+    public void setInspectUserCn(String inspectUserCn) {
+        this.inspectUserCn = inspectUserCn;
+    }
+
+    public String getRectifyUserCn() {
+        return rectifyUserCn;
+    }
+
+    public void setRectifyUserCn(String rectifyUserCn) {
+        this.rectifyUserCn = rectifyUserCn;
+    }
+
+    public Date getRectifyTime() {
+        return rectifyTime;
+    }
 
-	 public void setRectifyTime(Date rectifyTime) {
-		 this.rectifyTime = rectifyTime;
-	 }
+    public void setRectifyTime(Date rectifyTime) {
+        this.rectifyTime = rectifyTime;
+    }
 
-	public ProblemLibraryVo() {
-		super();
-	}
+    public ProblemLibraryVo() {
+        super();
+    }
 }
Index: admin/src/main/resources/com/info/admin/dao/impl/ProblemLibraryDao.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/resources/com/info/admin/dao/impl/ProblemLibraryDao.xml	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/resources/com/info/admin/dao/impl/ProblemLibraryDao.xml	(date 1542770655671)
@@ -142,6 +142,8 @@
 		<result column="inspect_content" property="inspectContent" />
 		<result column="inspect_user" property="inspectUser" />
 		<result column="rectify_user" property="rectifyUser" />
+		<result column="inspect_user_cn" property="inspectUserCn" />
+		<result column="rectify_user_cn" property="rectifyUserCn" />
 		<result column="rectify_time" property="rectifyTime" />
 		<result column="rectifyTime" property="rectifyTimeStr" />
     </resultMap>
@@ -168,13 +170,13 @@
 			    and seq =  #{entity.seq} 
 		    </if>
 		    <if test=" entity.title != null">
-			    and title =  #{entity.title} 
+				and title like concat('%',#{entity.title},'%')
 		    </if>
 		    <if test=" entity.type != null">
 			    and type =  #{entity.type} 
 		    </if>
 		    <if test=" entity.inspectContent != null">
-			    and inspect_content =  #{entity.inspectContent} 
+				and inspect_content like concat('%',#{entity.inspectContent},'%')
 		    </if>
 		    <if test=" entity.inspectUser != null">
 			    and inspect_user =  #{entity.inspectUser} 
@@ -194,7 +196,9 @@
           *
            ,date_format(create_time,'%Y-%m-%d %H:%i:%s') as createTime 
            ,date_format(update_time,'%Y-%m-%d %H:%i:%s') as updateTime 
-           ,date_format(rectify_time,'%Y-%m-%d %H:%i:%s') as rectifyTime 
+           ,date_format(rectify_time,'%Y-%m-%d %H:%i:%s') as rectifyTime
+			,(select name from staff_info where staff_info.staff_id = inspect_user ) as inspect_user_cn
+			,(select name from staff_info where staff_info.staff_id = rectify_user ) as rectify_user_cn
           from problem_library
         <include refid="filterCondition"></include>
 		order by seq
@@ -212,7 +216,9 @@
          * 
            ,date_format(create_time,'%Y-%m-%d %H:%i:%s') as createTime 
            ,date_format(update_time,'%Y-%m-%d %H:%i:%s') as updateTime 
-           ,date_format(rectify_time,'%Y-%m-%d %H:%i:%s') as rectifyTime 
+           ,date_format(rectify_time,'%Y-%m-%d %H:%i:%s') as rectifyTime
+			,(select name from staff_info where staff_info.staff_id = inspect_user ) as inspect_user_cn
+			,(select name from staff_info where staff_info.staff_id = rectify_user ) as rectify_user_cn
          from problem_library
         <include refid="filterCondition"></include>
         order by seq LIMIT #{offset},#{pageSize}
@@ -224,7 +230,9 @@
 		 * 
            ,date_format(create_time,'%Y-%m-%d %H:%i:%s') as createTime 
            ,date_format(update_time,'%Y-%m-%d %H:%i:%s') as updateTime 
-           ,date_format(rectify_time,'%Y-%m-%d %H:%i:%s') as rectifyTime 
+           ,date_format(rectify_time,'%Y-%m-%d %H:%i:%s') as rectifyTime
+			,(select name from staff_info where staff_info.staff_id = inspect_user ) as inspect_user_cn
+			,(select name from staff_info where staff_info.staff_id = rectify_user ) as rectify_user_cn
 		 from problem_library where supplier_id= #{supplierId}
 	</select>
 </mapper>	
Index: admin/src/main/webapp/WEB-INF/jsp/decorators/addHeader.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/webapp/WEB-INF/jsp/decorators/addHeader.jsp	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/webapp/WEB-INF/jsp/decorators/addHeader.jsp	(date 1542769592230)
@@ -94,4 +94,14 @@
         });
     }
 
+    //id开头的input 标签
+    var getValLetters = function (lableId) {
+        var input = $("input[id^='"+lableId+"']");
+        var vals = new Array();
+        $(input).each(function (i,n) {
+            vals.push($(n).val());
+        });
+        return vals;
+    };
+
 </script>
Index: admin/src/main/webapp/WEB-INF/jsp/problemlibrary/addProblemLibrary.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/webapp/WEB-INF/jsp/problemlibrary/addProblemLibrary.jsp	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/webapp/WEB-INF/jsp/problemlibrary/addProblemLibrary.jsp	(date 1542771047199)
@@ -11,20 +11,11 @@
  		<form id="saleForm" class="layui-form" style="margin-top:30px;">
  			<input  type="hidden" id="supplierId" name="supplierId" value="${problemLibrary.supplierId}"/>
 			<div class='layui-form-item'>
-     			<div class="layui-inline">
-		        	<label class="layui-form-label">排序号</label>
-		        	<div class="layui-input-inline">
-		        		<input type="text" id="seq" name="seq" placeholder="请输入排序号"  value="${problemLibrary.seq }" class="layui-input"/>
-		        		<span style="color: red" id="s-seq"></span>
-		     		</div>
-     			</div>
-     			<div class="layui-inline">
-		        	<label class="layui-form-label">标题</label>
-		        	<div class="layui-input-inline">
-		        		<input type="text" id="title" name="title" placeholder="请输入标题"  value="${problemLibrary.title }" class="layui-input"/>
-		        		<span style="color: red" id="s-title"></span>
-		     		</div>
-     			</div>
+				<label class="layui-form-label">标题</label>
+				<div class="layui-input-block">
+					<input type="text" id="title" name="title" style="width: 80%;" placeholder="请输入标题"  value="${problemLibrary.title }" class="layui-input"/>
+					<span style="color: red" id="s-title"></span>
+				</div>
 			</div>
 			<div class='layui-form-item'>
      			<div class="layui-inline">
@@ -34,25 +25,32 @@
 		        		<span style="color: red" id="s-type"></span>
 		     		</div>
      			</div>
+				<div class="layui-inline">
+					<label class="layui-form-label">排序号</label>
+					<div class="layui-input-inline">
+						<input type="text" id="seq" name="seq" placeholder="请输入排序号"  value="${problemLibrary.seq }" class="layui-input"/>
+						<span style="color: red" id="s-seq"></span>
+					</div>
+				</div>
 			</div>
 			<div class='layui-form-item'>
 		        <label class="layui-form-label">检查内容</label>
 				<div class="layui-input-block">
-					<textarea placeholder="请输入检查内容" style="width: 90%;" class="layui-textarea"  id="inspectContent" name = "inspectContent" lay-verify="content">${problemLibrary.inspectContent}</textarea>
+					<textarea placeholder="请输入检查内容" style="width: 80%;" class="layui-textarea"  id="inspectContent" name = "inspectContent" lay-verify="content">${problemLibrary.inspectContent}</textarea>
 					<span style="color: red" id="s-inspectContent"></span>
 				</div>
 			</div>
 			<div class='layui-form-item'>
      			<div class="layui-inline">
 		        	<label class="layui-form-label">检查人编号</label>
-		        	<div class="layui-input-inline">
+		        	<div class="layui-input-inline" id="div-inspectUser">
 		        		<input type="text" id="inspectUser" name="inspectUser" placeholder="请输入检查人编号"  value="${problemLibrary.inspectUser }" class="layui-input"/>
 		        		<span style="color: red" id="s-inspectUser"></span>
 		     		</div>
      			</div>
      			<div class="layui-inline">
 		        	<label class="layui-form-label">整改人编号</label>
-		        	<div class="layui-input-inline">
+		        	<div class="layui-input-inline" id="div-rectifyUser">
 		        		<input type="text" id="rectifyUser" name="rectifyUser" placeholder="请输入整改人编号"  value="${problemLibrary.rectifyUser }" class="layui-input"/>
 		        		<span style="color: red" id="s-rectifyUser"></span>
 		     		</div>
@@ -66,9 +64,8 @@
 		            	<span style="color: red" id="s-rectifyTime"></span>
 		        	</div>
      			</div>
-	   </div>
+	  		</div>
 
-	 		</div>
 	 		<div class="layui-input-block" style="margin-top:30px;margin-left:40%;">
 	 			<input type="button" class="layui-btn" onclick="saveData()" value="确定"/>
        			<button class="layui-btn" name="cancleSubmit" >取消</button>
@@ -76,6 +73,72 @@
   		</form>
 	</body>
 
+	<script type="text/javascript">
+
+        $(function () {
+			 $("#inspectUser").combotree({
+				 url: '/admin/staffInfo/staffInfoTree',
+				 // multiple : true,//设置可以多选，显示多选框，不设置不会出现多选框
+				 data : [{},{},{}],//数据省略
+				 required: true,
+				 checkbox : true,//显示多选框
+				 onlyLeafCheck : true,//只在叶子节点显示多选框
+				 onBeforeSelect : function(node){
+					 if(!$(this).tree("isLeaf", node.target)){//如果不是叶子节点，不让选择
+						 return false;
+					 }
+				 },
+				 onBeforeCheck : function(node, checked){//控制只能选一项
+					 if(checked){//当前为选中操作
+						 var nodes = $(this).tree("getChecked");
+                         $("#div-inspectUser").append("<input type='hidden' id='inspectUser-"+node.id+"' name='inspectUser-"+node.id+"' value='"+node.text+"'/>");
+						 //控制只能选一项，选中某一项后后面不能再勾选
+						 if(nodes.length == 0){
+							 return true;
+						 }else{
+							 return false;
+						 }
+					 }else{//当前为取消选中操作
+                         $("#inspectUser-"+node.id+"").remove();
+						 return true;
+					 }
+				 }
+			 });
+
+            $("#rectifyUser").combotree({
+                url: '/admin/staffInfo/staffInfoTree',
+                // multiple : true,//设置可以多选，显示多选框，不设置不会出现多选框
+                idField:'staffId',           //定义标识树节点的键名字段
+                treeField:'name',       //定义树节点的字段
+                data : [{},{},{}],//数据省略
+                required: true,
+                checkbox : true,//显示多选框
+                onlyLeafCheck : true,//只在叶子节点显示多选框
+                onBeforeSelect : function(node){
+                    if(!$(this).tree("isLeaf", node.target)){//如果不是叶子节点，不让选择
+                        return false;
+                    }
+                },
+                onBeforeCheck : function(node, checked){//控制只能选一项
+                    if(checked){//当前为选中操作
+                        var nodes = $(this).tree("getChecked");
+                        $("#div-rectifyUser").append("<input type='hidden' id='rectifyUser-"+node.id+"' name='rectifyUser-"+node.id+"' value='"+node.text+"'/>");
+                        //控制只能选一项，选中某一项后后面不能再勾选
+                        if(nodes.length == 0){
+                            return true;
+                        }else{
+                            return false;
+                        }
+                    }else{//当前为取消选中操作
+                        $("#rectifyUser-"+node.id+"").remove();
+                        return true;
+                    }
+                }
+            });
+        });
+
+	</script>
+
 	<script type="text/javascript">
 		var reqUpdateAndAddUrl = "/admin/problemLibrary/insertAndUpdate";
 	</script>
@@ -103,18 +166,27 @@
 	    var rectifyUser = $("#rectifyUser").val();
 	    var rectifyTime = $("#rectifyTime").val();
 
+        var inspectUsers = $('#inspectUser').combotree('getValues');
+        var inspectUserCn = getValLetters("inspectUserCn-");
+        var rectifyUsers = $('#rectifyUser').combotree('getValues');
+        var rectifyUserCn = getValLetters("rectifyUserCn-");
+
         var requestData={
             "supplierId":supplierId,
             "seq":seq,
             "title":title,
             "type":type,
             "inspectContent":inspectContent,
-            "inspectUser":inspectUser,
+            "inspectUsers":inspectUsers,
+            "rectifyUsers":rectifyUsers,
+			"inspectUser":inspectUser,
             "rectifyUser":rectifyUser,
             "rectifyTimeStr":rectifyTime
         };
         updateAndAdd(requestData);
     }
+
+
 	
 </script>
 </html>				 
Index: admin/src/main/webapp/WEB-INF/jsp/problemlibrary/listProblemLibrary.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/webapp/WEB-INF/jsp/problemlibrary/listProblemLibrary.jsp	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/webapp/WEB-INF/jsp/problemlibrary/listProblemLibrary.jsp	(date 1542802414870)
@@ -61,7 +61,24 @@
 	                      <input type="hidden" name="pageSize" id="pageSize" value="${paginator.pageRecord}">		           	 
 			              <div class="box-body">
 			                 <div class="form-group">
-
+								 <label for="title" class="col-sm-2 control-label">标题：</label>
+								 <div class="col-xs-2">
+									 <input type="text" class="form-control" id="title" name="title" value="${problemLibrary.title}" placeholder="请输入标题">
+								 </div>
+								 <label for="inspectContent" class="col-sm-2 control-label">检查内容：</label>
+								 <div class="col-xs-2">
+									 <input type="text" class="form-control" id="inspectContent" name="inspectContent" value="${problemLibrary.inspectContent}" placeholder="请输入检查内容"/>
+								 </div>
+								 <label for="type" class="col-sm-2 control-label">问题类型</label>
+								 <div class="col-xs-2">
+									 <input type="text" class="form-control" id="type" name="type" value="${problemLibrary.type}" placeholder="请输入问题类型"/>
+
+									 <%--<select id="status" name="status" class="form-control" style="width:150px;">
+										 <option value="" selected>全部</option>
+										 <option value="1" <c:if test="${problemLibrary.type == 1}">selected</c:if>>启用</option>
+										 <option value="-1" <c:if test="${problemLibrary.type == -1}">selected</c:if>>禁用</option>
+									 </select>--%>
+								 </div>
 			                 </div>
 			                 <div class="box-footer">
 			                 	<button onclick='refreshTheCurrentPage()' class="btn btn-info pull-left">查询</button>
@@ -84,11 +101,9 @@
 			                     <th field="title"  >标题</th>
 			                     <th field="type"  >问题类型</th>
 			                     <th field="inspectContent"  >检查内容</th>
-			                     <th field="inspectUser"  >检查人编号</th>
-			                     <th field="rectifyUser"  >整改人编号</th>
+			                     <th field="inspectUserCn"  >检查人</th>
+			                     <th field="rectifyUserCn"  >整改人</th>
 			                     <th field="rectifyTime"  type='date'>整改时间</th>
-			                     <th field="createTime"  type='date'>创建时间</th>
-			                     <th field="updateTime"  type='date'>修改时间</th>
 							     <th field="seq"  >排序号</th>
 				                 <th field="sys_opt">操作</th>
 				              </tr>
@@ -100,11 +115,9 @@
 					             <td>${r.title}</td>
 					             <td>${r.type}</td>
 					             <td>${r.inspectContent}</td>
-					             <td>${r.inspectUser}</td>
-					             <td>${r.rectifyUser}</td>
+					             <td>${r.inspectUserCn}</td>
+					             <td>${r.rectifyUserCn}</td>
 				                 <td><fmt:formatDate value="${r.rectifyTime }" pattern="yyyy-MM-dd HH:mm:ss"/></td>
-				                 <td><fmt:formatDate value="${r.createTime }" pattern="yyyy-MM-dd HH:mm:ss"/></td>
-				                 <td><fmt:formatDate value="${r.updateTime }" pattern="yyyy-MM-dd HH:mm:ss"/></td>
 								 <td>${r.seq}</td>
 						        <td>
 						         <div class="site-demo-button" >
Index: admin/src/main/webapp/WEB-INF/jsp/problemlibrary/listProblemLibraryDesktop.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/webapp/WEB-INF/jsp/problemlibrary/listProblemLibraryDesktop.jsp	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/webapp/WEB-INF/jsp/problemlibrary/listProblemLibraryDesktop.jsp	(date 1542770744784)
@@ -62,7 +62,24 @@
 	                      <input type="hidden" name="pageSize" id="pageSize" value="${paginator.pageRecord}">		           	 
 			              <div class="box-body">
 			                 <div class="form-group">
-
+								 <label for="userName" class="col-sm-2 control-label">标题：</label>
+								 <div class="col-xs-2">
+									 <input type="text" class="form-control" id="title" name="title" value="${problemLibrary.title}" placeholder="请输入标题">
+								 </div>
+								 <label for="phone" class="col-sm-2 control-label">检查内容：</label>
+								 <div class="col-xs-2">
+									 <input type="text" class="form-control" id="inspectContent" name="inspectContent" value="${problemLibrary.inspectContent}" placeholder="请输入检查内容"/>
+								 </div>
+								 <label for="status" class="col-sm-2 control-label">问题类型</label>
+								 <div class="col-xs-2">
+									 <input type="text" class="form-control" id="type" name="type" value="${problemLibrary.type}" placeholder="请输入问题类型"/>
+
+										 <%--<select id="status" name="status" class="form-control" style="width:150px;">
+                                             <option value="" selected>全部</option>
+                                             <option value="1" <c:if test="${problemLibrary.type == 1}">selected</c:if>>启用</option>
+                                             <option value="-1" <c:if test="${problemLibrary.type == -1}">selected</c:if>>禁用</option>
+                                         </select>--%>
+								 </div>
 			                 </div>
 			                 <div class="box-footer">
 			                 	<button onclick='refreshTheCurrentPage()' class="btn btn-info pull-left">查询</button>
@@ -86,8 +103,8 @@
 			                    <th field="title"  >标题</th>
 			                    <th field="type"  >问题类型</th>
 			                    <th field="inspectContent"  >检查内容</th>
-			                    <th field="inspectUser"  >检查人编号</th>
-			                    <th field="rectifyUser"  >整改人编号</th>
+			                    <th field="inspectUserCn"  >检查人</th>
+			                    <th field="rectifyUserCn"  >整改人</th>
 			                    <th field="rectifyTime"  type='date'>整改时间</th>
 
 				                <th field="sys_opt">操作</th>
@@ -101,8 +118,8 @@
 					            <td>${r.title}</td>
 					            <td>${r.type}</td>
 					            <td>${r.inspectContent}</td>
-					            <td>${r.inspectUser}</td>
-					            <td>${r.rectifyUser}</td>
+					            <td>${r.inspectUserCn}</td>
+					            <td>${r.rectifyUserCn}</td>
 				                <td><fmt:formatDate value="${r.rectifyTime }" pattern="yyyy-MM-dd HH:mm:ss"/></td>
 
 						        <td>
Index: admin/src/main/webapp/WEB-INF/jsp/releaseinfo/addReleaseInfo.jsp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- admin/src/main/webapp/WEB-INF/jsp/releaseinfo/addReleaseInfo.jsp	(revision 700d0bca20c23462c394006aa31a0f0f40c5613f)
+++ admin/src/main/webapp/WEB-INF/jsp/releaseinfo/addReleaseInfo.jsp	(date 1542769613001)
@@ -186,7 +186,7 @@
 			var content = $("#content").val();
 			var releaseUser = $('#releaseUser').val();
 			var receiveUser = $('#receiveUser').combotree('getValues');
-			var receiveUserCn = getVals("receiveUserCn-");
+			var receiveUserCn = getValLetters("receiveUserCn-");
 			var releaseType  = $("#releaseType").val();
 			var requestData={
 				"releaseId":releaseId,
@@ -203,16 +203,5 @@
 			updateAndAdd(requestData);
 		}
 
-		//id开头的input 标签
-        var getVals = function (lableId) {
-            var input = $("input[id^='"+lableId+"']");
-            var vals = new Array();
-            $(input).each(function (i,n) {
-                vals.push($(n).val());
-            });
-            return vals;
-        };
-
-
 </script>
 </html>				 
